maxime-rainville/silverstripe-react
# Feature/react admin default 404 component

#Description
New Default Component. It will be shown if user hasn't defined Component name in getComponent method.
New ErrorComponent. It will be shown if user defined wrong Component name in getComponent method and Component doesn't exist.

#Test
- Run `yarn dev`
- Open `\admin\innovation\` page
- User should see "Innovation week" title
- In `InnovationWeek.php` change Component name in getComponent method to the new name, for example 'Test'.
- Run `composer dev-build`
- Open `\admin\innovation\` page
- User should see "Component NOT FOUND" title
- In `InnovationWeek.php` replace Component name in getComponent method to the empty string.
- Run `composer dev-build`
- Open `\admin\innovation\` page
- User should see "Default component" title
